action.yaml

```diff
@@ -42,8 +42,82 @@ runs:
     - name: Install tools
       # This will skip the (slow) pyenv install if we triggered the above
       uses: open-turo/action-setup-tools@v1
+    - name: Install @commitlint/cli
+      shell: bash
+      run: |
+        npm install --no-save --no-package-lock --no-audit --no-fund \
+          --prefix "$RUNNER_TEMP" "@commitlint/cli"
+        echo "$RUNNER_TEMP/node_modules/.bin" >> $GITHUB_PATH
+        echo "NODE_PATH=$RUNNER_TEMP/node_modules" >> $GITHUB_ENV
+    - name: Use default conventional commit config
+      if: inputs.turo-conventional-commit
+      shell: bash
+      run: |
+        # Write default commitlint config in the workspace.
+        echo "extends: [\"@open-turo/commitlint-config-conventional\"]" \
+          > "${{ inputs.config-file }}"
+        # XXX(shakefu): We have to list @commitlint/cli here otherwise `npm
+        # install` will try to remove it from the above step. There is no way to
+        # prevent this behavior that I'm aware of.
+        npm install --no-save --no-package-lock --no-audit --no-fund \
+          --prefix "$RUNNER_TEMP" \
+          "@open-turo/commitlint-config-conventional" \
+          "@commitlint/cli"
     - name: Commitlint
-      uses: ./commitlint
```

Comment:
Delete this file too?

Reply:
Keeping the file so it works as a sub-action still.